data/test/scenarios/wml_tests/UnitsWML/AbilitiesWML/ability_cycle_recursion_by_id.cfg

```diff
@@ -0,0 +1,58 @@
+#textdomain wesnoth-test
+
+#####
+# API(s) being tested: [filter]ability_id_active=
+##
+# Actions:
+# Alice and Bob are both of type Test Melee Quintain.
+# Give Alice's leadership_test_recursion ability.
+# leadership_test_recursion active if itself active
```

Comment:
Give Alice and Bob a leadership_test_recursion ability, which is active only if it is itself active.

Reply:
fixed